__test__/integration/adapters/collateralized_simple_interest_loan_adapter/scenarios/successful_seize_collateral_scenarios.ts

```diff
@@ -0,0 +1,15 @@
+import * as _ from "lodash";
+import { BigNumber } from "bignumber.js";
+
+import { defaultArgs, SeizeCollateralScenario } from "./";
+
+const defaultArgsWithNoGracePeriod = _.clone(defaultArgs);
+defaultArgsWithNoGracePeriod.collateralTerms.gracePeriodInDays = new BigNumber(0);
+
+export const SUCCESSFUL_SEIZE_COLLATERAL_SCENARIOS: SeizeCollateralScenario[] = [
+    {
+        ...defaultArgsWithNoGracePeriod,
+        description: "when there is no grace period the debt has not been paid",
+        debtRepaid: false,
+    },
```

Comment:
We should add test cases for when the grace period > 0 but has nonetheless  lapsed.